File: raeunlee/week_36/SOFT_비밀메뉴.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(int argc, char** argv) {
    int m, n, k;
    cin >> m >> n >> k;

    vector<int> secret(m);
    for (int i = 0; i < m; i++) {
        cin >> secret[i];
    }

    vector<int> button(n);
    for (int i = 0; i < n; i++) {
        cin >> button[i];
    }

    int count = 0;

    if (secret.size() > button.size()) {
        count = 0;
    } else {
        for (int i = 0; i <= button.size() - secret.size(); i++) {
            vector<int> tmp(button.begin() + i, button.begin() + i + secret.size());
            if (tmp == secret) {
                count += 1;
            }
        }
    }

    if (count > 0) {
        cout << "secret";
    } else {
        cout << "normal";
    }

    return 0;
}

File: raeunlee/week_36/SOFT_비밀메뉴.py
import sys

m, n, k = map(int, input().split())
secret = list(map(int, input().split()))
button = list(map(int, input().split()))
count = 0
if len(button) < len(secret):
    count = 0
else:
    for i in range(len(button)-len(secret)+1):
        tmp = button[i:i+len(secret)]
        if tmp == secret:
            count += 1
if count > 0:
    print("secret")
else:
    print("normal")

            
        